tests/lib/rules/alphabetical-paths.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * @fileoverview Tests for `alphabetical-paths` rule
+ * @author Ondřej Brejla <ondrej@brejla.cz>
+ */
+
+"use strict";
+
+//------------------------------------------------------------------------------
+// Requirements
+//------------------------------------------------------------------------------
+
+var RuleTester = require("eslint").RuleTester,
+    fixtures = require("../../fixtures"),
+    rule = require("../../../lib/rules/alphabetical-paths");
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+
+var ERROR = {
+    message: "Required paths are not in alphabetical order.",
+    type: "CallExpression"
+};
+
+var ruleTester = new RuleTester();
+
+ruleTester.run("alphabetical-paths", rule, {
+
+    valid: [
+        fixtures.ALPHABETICAL_PATHS_NO_PATH,
+        fixtures.ALPHABETICAL_PATHS_NO_PATH_EMPTY_ARRAY,
+        fixtures.ALPHABETICAL_PATHS_ONE_PATH_IN_ARRAY,
+        fixtures.ALPHABETICAL_PATHS_MORE_PATHS_IN_ARRAY,
+
+        fixtures.ALPHABETICAL_PATHS_REQUIRE_NO_PATH,
+        fixtures.ALPHABETICAL_PATHS_REQUIRE_NO_PATH_EMPTY_ARRAY,
+        fixtures.ALPHABETICAL_PATHS_REQUIRE_ONE_PATH_IN_ARRAY,
+        fixtures.ALPHABETICAL_PATHS_REQUIRE_MORE_PATHS_IN_ARRAY,
+
+        {
+            code: fixtures.ALPHABETICAL_PATHS_BASENAME_VALID_ORDER,
+            options: [ "basename" ]
+        }
+    ],
```

Comment:
I tried these fixtures in a valid section and it passed:

fixtures.CJS_WITH_RETURN,
        fixtures.CJS_WITH_EXPORTS,
        fixtures.CJS_WITH_MODULE_EXPORTS,
        fixtures.CJS_WITH_FUNC_EXPR,
        fixtures.CJS_WITH_INVALID_EXPORTS,
        fixtures.DYNAMIC_AMD_REQUIRE_WITH_ERRBACK,
        fixtures.DYNAMIC_AMD_REQUIREJS_WITH_ERRBACK

Should I add them to tests?